README.md

```diff
@@ -431,6 +432,12 @@ Note that the output values will be random since the input values are random.
 
 <br>
 
+## ONNX Operators supported by CPU <a id="cpu-ops"></a>
+
+For the most up to date list, see [Supported ONNX Operation for Target CPU](https://github.com/onnx/onnx-mlir/blob/main/docs/SupportedONNXOps-cpu.md) in the onnx-mlir repository.
```

Comment:
The zDLC images in ICR are released on a cadence, not on every onnx-mlir commit. This should link to the onnx-mlir docs that match this version of zDL. So this should be something to the effect of "the list of supported Operators for this version of zDLC".